### ISSUE TITLE
Improve the name for the SINK ADDRESS upon registering a Namespace

### ISSUE THREAD

**Issue by r3n3x** [NemProject/NanoWallet#37]
When you order a Namespace it shows a "SINK ADDRESS" which made me wonder 
"why would i want to register a root namespace aka Alias for that address?"
Well, to try thing out - I just did register and voilà I have my alias working for my main NEM address.
I suggest to rephrase in the NanoWallet design.
The term "SINK ADDRESS" for Namespace ordering is more confusing than helpful.

Suggestion: "Rental fee will be sent to:"
![screenshot from 2017-03-25 13-42-22](https://cloud.githubusercontent.com/assets/23291787/24319825/7f451e66-1162-11e7-8fe0-b339f128f225.png)

**Comment by r3n3x** [comment]
Kodtycoon comment:
i see where your coming from alright. it would probably be better to simply remove the "sink address" from the display entirely and just have a namespace registry fee field showing the cost on its own. then there would be no confusion, as people will just understand thats the fee you have to pay.

**Comment by saulgray** [comment]
I think we can just fix this by adding a tooltip. When someone hovers over, they will see "This is the address rental fees get paid to."

**Comment by cryptoBeliever** [comment]
I agree with comment https://github.com/NemProject/NanoWallet/issues/37#issuecomment-289191395. A sink address presentation is not needed here.
